python/ray/serve/config.py

```diff
@@ -6,6 +6,7 @@
 import pydantic
 from google.protobuf.json_format import MessageToDict
 from pydantic import BaseModel, NonNegativeFloat, PositiveInt, validator
+from pydantic.types import NonNegativeInt, PositiveFloat
```

Comment:
the import is weird, can we just import NonNegativeInt, PositiveFloat from pydantic instead of pydantic.types

Reply:
Good call, I revised the imports.